chapters/improvements/sections/world_improvements.tex

```diff
@@ -0,0 +1,17 @@
+\section{World improvements}
+The world could be improved in multiple ways.
+One such way would be to add shadows to the game.
+Right now nothing casts any shadows but it would be interesting to see what shadows look like in non-Euclidean geometries.
+
+Another improvement to the world would be to add clouds.
+The sky is very empty during the day with only the suns visible.
+Its better during the night but nevertheless clouds would make it look even better.
```

Comment:
Something along the lines of "During the night, due to the stars and other celestial bodies, the sky looks much better"?